example/src/main/java/ably/io/tutorial/ExampleActivity.java

```diff
@@ -49,4 +70,22 @@ public void onMessage(Message messages) {
             }
         });
     }
+
+    /* Add AblyException to method signature as Channel#publish method can throw one */
+    private void publishMessage(String message) throws AblyException {
+
+        channel.publish("update", message, new CompletionListener() {
+            @Override
+            public void onSuccess() {
+                /* Print success message when message is sent */
+                Log.v(getClass().getCanonicalName(), "Message sent");
```

Comment:
> On Android it's probably simplest to show a Toast but I'm not sure if we're trying to keep the code so that it is usable both with Android and JRE.

No, I think we should have different tutorials for JRE and Android